buildSrc/src/main/kotlin/config/AndroidLibraryConfig.kt

```diff
@@ -24,15 +25,15 @@ fun AndroidLibraryExtension.setDefaults(generateBuildConfig: Boolean = false) {
         this.buildConfig = generateBuildConfig
     }
     sourceSets.getByName("main") {
-        manifest.srcFile("src/androidMain/AndroidManifest.xml")
+        if (project.file("src/androidMain/AndroidManifest.xml").exists()) {
+            manifest.srcFile("src/androidMain/AndroidManifest.xml")
+        }
         res.srcDir("src/androidMain/res")
     }
     sourceSets.getByName("debug") {
-        manifest.srcFile("src/androidDebug/AndroidManifest.xml")
         res.srcDir("src/androidDebug/res")
     }
     sourceSets.getByName("release") {
-        manifest.srcFile("src/androidRelease/AndroidManifest.xml")
```

Comment:
Why did you remove these?
I have this for the cases where I'd need buildType specific manifest.
Does it currently causes issues with auto-manifest?
Or could it cause issues if I were to actually have buildTypes specific issues while using auto-manifest?

Reply:
It does not cause any issues. I was gonna comment here and forgot. I actually checked and there is no such manifest file for debug build types. That's why I did this.  I should bring them back.